test/fixtures/integration/extinf.js

```diff
@@ -19,7 +19,8 @@ module.exports = {
       },
       duration: 10,
       timeline: 0,
-      uri: 'hls_450k_video.ts'
+      uri: 'hls_450k_video.ts',
+      title: ';asljasdfii11)))00,'
```

Comment:
this doesn't seem right. Though, that's down to how it's being parsed and not a blocker for this PR.